BuildResidentialHPXML/resources/constants.rb

```diff
@@ -48,12 +48,4 @@ def self.NumHoursInYear(is_leap_year = false)
     num_hours_in_year = num_days_in_year * 24
     return num_hours_in_year.to_f
   end
-
-  def self.PeakFlowRate
```

Comment:
Hmm, I thought there was a specific reason for using these. @rajeee would probably know.

Reply:
I spoke with @rajeee and my understanding is that Constants provide values in the schedule csv that have some physical basis (i.e., power or flowrate values match the raw data that the schedules are derived from). However, that is a bit misleading in the context of the simulation because the schedule values are only used to calculate _when_ energy is used, not _how much_. It's also confusing to mix units between columns when, at the end of the day, they are treated as fractional inputs to the simulation engine.

Additionally, the values should have no bearing on the simulation results, but testing showed that they actually did. This is because other code in our model generation (specifically the dryer vent model) assumes that schedules are normalized to 1. In EnergyPlus, that convention makes the most sense because then the equipment design levels or peak flow rates are closer to reality.

So in summary, I think it's more intuitive and less error-prone to use normalized columns everywhere in the context of EnergyPlus.